02_getting_started.asciidoc

```diff
@@ -24,6 +24,8 @@ The below is an inexhaustive list (in alphanumerical order):
 ====
 Note that if using Lightning explorers, just like in existing block explorers,
 the privacy can be a concern because, if users are careless, the website may track their IP addresses and collect their behavior records (for example, the nodes users are interested in).
+To reclaim your privacy and stay anonymous online, it is recommended to use The Onion Router (TOR) Browser footnote:[TOR Project can help you to defend yourself against tracking and surveillance and circumvent censorship  (https://www.torproject.org/download/).]
```

Comment:
yes and no. It seems kind of offtopic in the context of our book to give recommendations to people how to behave while browsing the web.